api/spec/services/short_service_calculator_spec.rb

```diff
@@ -11,6 +11,26 @@
     let(:required_service_days) { 26 }
     let(:calculated_ending_day) { short_service_calculator.calculate_ending_date(required_service_days) }
 
+    context 'with company holidays' do
```

Comment:
You just test a case where no paid vacation is available, right?
Maybe add a spec where you can compensate

Reply:
There are no paid vacations in a Service with less than 180 days